doc/13_1_setup_spark_shell.md

```diff
@@ -0,0 +1,32 @@
+# Documentation
+
+## Setting up Cassandra
+
+The easiest way to get quickly started with Cassandra is to follow the instructions provided by 
+[Datastax](http://docs.datastax.com/en/cassandra/2.1/cassandra/install/install_cassandraTOC.html)
+
```

Comment:
@RussellSpitzer  should we mention a docker alternative? That's a quite low-threshold way to quickly build up a playground. Al Tobey maintains one image: https://github.com/tobert/cassandra-docker